app/assets/javascripts/sequence_show.js

```diff
@@ -512,3 +535,33 @@ function init_sequence_show(data) {
         }
     }
 }
+
+function toggle_column(col) {
```

Comment:
http://stackoverflow.com/a/8862711/865696


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3473) by @bmesuere on Mon Nov 09 2015 at 22:29._